sympy/polys/ring_series.py

```diff
@@ -606,6 +736,9 @@ def rs_exp(p, x, prec):
     1/6*x**6 + 1/2*x**4 + x**2 + 1
     """
     ring = p.ring
+    index = ring.gens.index(x)
+    #n = max(p, key=lambda k: k[index])[index]
+    #prec = _check_precision(prec, n)
```

Comment:
#certik If you uncomment these two lines, you can check if the method works with `rs_exp`. I haven't yet decided what to do if the degree or the precision is negative.